src/pfe/portal/modules/Templates.js

```diff
@@ -454,17 +454,33 @@ function isRepo(obj) {
 async function doesUrlPointToIndexJson(inputUrl) {
   const url = new URL(inputUrl); // Throws error if `inputUrl` is not a valid url
 
-  const options = {
-    host: url.host,
-    path: url.pathname,
-    method: 'GET',
+  let templateSummariesText = '[]';
+  if ( url.protocol === 'file:' ) {
+    try {
+      if ( fs.existsSync(url.pathname) ) {
+        let data = fs.readFileSync(url.pathname, "utf-8");
```

Comment:
Could this be changed to `await fs.readFile(url.pathname, "utf-8")` so it doesn't block? (Also fs.existsSync can be changed.)

Reply:
@hhellyer thanks, changed to await on async fs calls.

Reply:
@lentheivendra - Thanks, I think we had a linter rule that warned about using sync methods. I've made a note to check whether we're running the linter on Jenkins now.